### PR TITLE
Creates unit tests

### PR DESCRIPTION
**Pull request by robertbullen**
Addresses issue #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +80,15 @@ Destroy the instance of an invisible grecaptcha.
 
 See the [contributing file](CONTRIBUTING.md).
 
+## Testing
```
Comment (thiamsantos) [inline README.md]: Could you move this instructions to the contributing file?

File: src/tests/test.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+const path = require('path')
+const url = require('url')
+const puppeteer = require('puppeteer')
+const serveHandler = require('serve-handler')
+const testKeys = require('recaptcha-test-keys')
+const Grecaptcha = require('grecaptcha')
+
+// Encapsulate the asynchronous starting and stopping of a simple HTTP server at http://localhost:3000. This is needed
+// because reCAPTCHAs don't work from within pages loaded directly from the file system.
+const server = {
```
Comment (thiamsantos) [inline src/tests/test.js]: Could you move these objects (`api`, `browser` and `server`) to a test-helper file? Leaving on the test file only the assertions?

File: src/tests/test.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (thiamsantos) [inline src/tests/test.html]: Could you move the tests to its own folder? From `src/tests` to `test`?

**Review by thiamsantos — CHANGES_REQUESTED** [review]
Awesome @robertbullen! :tada: 

I made a few suggestions but in general great work. Would you mind to add the `npm test` command to the travis config file?

**Comment by robertbullen** [comment]
Here's what the output looks like after running `npm test`:

```
$ npm test

> invisible-grecaptcha@1.0.3 pretest
> npm run lint && npm run build


> invisible-grecaptcha@1.0.3 prelint
> prettier --write "src/**/*.js" --write rollup.config.js

src\index.js 73ms
src\tests\test.js 80ms
src\utils.js 25ms

> invisible-grecaptcha@1.0.3 lint 
> xo --fix


> invisible-grecaptcha@1.0.3 build
> microbundle build --target browser

Build output to dist:
        734 B: index.js
        736 B: index.m.js
        801 B: index.umd.js

> invisible-grecaptcha@1.0.3 test
> jest

 PASS  src/tests/test.js
  invisible-grecaptcha
    after starting the server and browser
      √ the test harness page loads successfully (73ms)
    after loading the distributable script
      √ the library 'window.invisibleGrecaptcha' is added (138ms)
      √ the library function 'window.invisibleGrecaptcha.execute()' is present (3ms)
      √ the library function 'window.invisibleGrecaptcha.destroy()' is present (3ms)
    when invoking the API
      √ 'window.invisibleGrecaptcha.execute()' adds a reCAPTCHA div to the page (17ms)
      √ 'window.invisibleGrecaptcha.execute()' returns a verifiable string token (162ms)
      √ 'window.invisibleGrecaptcha.destroy()' removes the reCAPTCHA div from the page (13ms)

Test Suites: 1 passed, 1 total
Tests:       7 passed, 7 total
Snapshots:   0 total
Time:        4.935s, estimated 11s
Ran all test suites.
```

**Comment by robertbullen** [comment]
All of your requested changes have been completed and the unit tests are now being executed during Travis CI process. Unless you have further suggestions, this PR is good to go.

File: .travis.yml [hunk 4]
```diff
@@ -2,5 +2,5 @@ language: node_js
 node_js:
   - '8'
 script:
-  - npm run lint
-  - npm run build
+  # Running the "test" script also runs `npm run lint` and `npm run build` as part of "pretest".
```
Comment (thiamsantos) [inline .travis.yml]: No need to add this comment.

File: test/test-helpers.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+      await this.page.tracing.stop()
+    }
+    if (!this._debug) {
+      await this._instance.close()
+    }
+  }
+}
+
+// Encapsulate all the library strings and their compositions for consistency.
+module.exports.api = {
```
Comment (thiamsantos) [inline test/test-helpers.js]: Could you add a single `module.exports` at the bottom of the file?

```js
module.exports = {
  api,
  browser,
  server
}
```

**Comment by robertbullen** [comment]
I made those last two changes you requested.

**Review by thiamsantos — APPROVED** [review]
@robertbullen thanks!

**Pull request merged**